lib/releaselib.sh

```diff
@@ -52,7 +52,7 @@ release::get_job_cache () {
   done < <(jq -r '.[] | select(.result == "SUCCESS") | select(.version != "") | [.version,.buildnumber] | "\(.[0]) \(.[1])"' $tempjson |\
    LC_ALL=C sort -rn -k2,2) |\
   while read version buildnumber; do
-    echo "JOB[$buildnumber]=$version"
+    [[ -n $buildnumber && -n $version ]] && echo "JOB[$buildnumber]=$version"
```

Comment:
I guess this is just going to skip over invalid lines in the intermediate `echo "$version $buildnumber"` output?